pipeline.cpp

```diff
@@ -73,14 +74,58 @@ void DrawLine(Buffer2D<PIXEL> & target, Vertex* const triangle, Attributes* cons
     // Your code goes here
 }
 
+/**************************************************************
+ * CROSS_PRODUCT
+ * Calculates the cross product of two vertecies.
+ *************************************************************/
```

Comment:
You implemented a cross product method. Nice job. My submission simply did all the calculations in the TestDrawTriangle method, I think.